common/utilities/integration_adapters_logger.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import sys
+import time
+
+AUDIT = 25
+
+
+def _add_audit_level_log():
+    """
+    Adds a new log level to the standard python logging module, this level is the `audit` level and is defined as
+    a high level than INFO but lower than  WARNING
+    """
+    logging.addLevelName(AUDIT, "AUDIT")
+
+    def audit(self, message, *args, **kws):
+        if self.isEnabledFor(AUDIT):
+            self._log(AUDIT, message, args, **kws)
+
+    logging.Logger.audit = audit
+
+
+def load_global_log_config():
+    """
+    A general method to load the overall config of the system, specifically it modifies the root handler to output
+    to sysout and sets the default log levels and format. This is expected to be called once at the start of a
+    system.
+    """
+    _add_audit_level_log()
+    logger = logging.getLogger()
+    logger.setLevel(logging.NOTSET)
+    handler = logging.StreamHandler(sys.stdout)
+    logging.Formatter.converter = time.gmtime
+    formatter = logging.Formatter('[%(asctime)s.%(msecs)03d] '
```

Comment:
I think we should include the `Z` suffix to indicate this timestamp is in UTC, as [otherwise the standard says the time is assumed to be local time](https://en.wikipedia.org/wiki/ISO_8601#Time_zone_designators).

```suggestion
    formatter = logging.Formatter('[%(asctime)s.%(msecs)03dZ] '
```